docker-compose.yml

```
@@ -329,16 +333,51 @@ services:
   cvat_clickhouse:
     container_name: cvat_clickhouse
     image: clickhouse/clickhouse-server:23.11-alpine
-    restart: always
+    restart: "always"
     environment:
       <<: *clickhouse-env
     networks:
       cvat:
         aliases:
           - clickhouse
     volumes:
-      - ./components/analytics/clickhouse/init.sh:/docker-entrypoint-initdb.d/init.sh:ro
       - cvat_events_db:/var/lib/clickhouse/
+    healthcheck:
+      test: ["CMD", "clickhouse-client", "--query", "SELECT 1"]
+      interval: 3s
+      timeout: 300s
+      retries: 3
+
+  cvat_clickhouse_init:
+    container_name: cvat_clickhouse_init
+    image: clickhouse/clickhouse-server:23.11-alpine
+    restart: "no"
+    environment:
+      <<: *clickhouse-env
+    depends_on:
+      cvat_clickhouse:
+        condition: service_healthy
+    networks:
+      cvat:
+        aliases:
+          - clickhouse
```

```suggestion
      - cvat:
```

It seems that we don't need define alias at all, since the service is not supposed to accept requests.